ci/terraform/oidc/re-verification-result.tf

```diff
@@ -0,0 +1,58 @@
+module "re_verification_result_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "re-verification-result_role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = [
+  ]
+}
+
+
+module "re_verification_result" {
+  source = "../modules/endpoint-module"
+
+  endpoint_name   = "re-verification-result"
+  path_part       = "re-verification-result"
+  endpoint_method = ["POST"]
+  environment     = var.environment
+
+  handler_environment_variables = {
+  }
+
+  handler_function_name = "uk.gov.di.authentication.frontendapi.lambda.ReverificationResultHandler::handleRequest"
+
+  rest_api_id      = aws_api_gateway_rest_api.di_authentication_frontend_api.id
+  root_resource_id = aws_api_gateway_rest_api.di_authentication_frontend_api.root_resource_id
+  execution_arn    = aws_api_gateway_rest_api.di_authentication_frontend_api.execution_arn
+
+  memory_size                 = lookup(var.performance_tuning, "re-verification-result", local.default_performance_parameters).memory
+  provisioned_concurrency     = 1
```

Comment:
Ah sorry, this is not what I meant. We need to set the values in the environment files. There should still be no provisioned concurrency in dev, like the default